environment.py

```diff
@@ -370,5 +370,13 @@ def env() -> Mapping[str, Optional[str]]:
         #
         # tdr:broad-jade-dev-data:dataset/hca_mvp
         # tdr:broad-jade-dev-data:snapshot/hca_mvp
-        'AZUL_TDR_TARGET': None
+        'AZUL_TDR_TARGET': None,
+
+        # Instance of the Terra Data Repository API to use for retrieving
+        # snapshot or dataset descrptions.
```

Comment:
```suggestion
        # The URL of the Terra Data Repository instance to index metadata from.
```

and wrap as necessary